packages/dashmate/configs/defaults/getMainnetConfigFactory.js

```diff
@@ -36,7 +36,42 @@ export default function getMainnetConfigFactory(homeDir, getBaseConfig) {
       },
       network: NETWORK_MAINNET,
       platform: {
-        enable: false,
+        tenderdash: {
+          p2p: {
+            seeds: [
+              {
+                id: '069639dfceec5f7c86257e6e9c46407c16ad1eab',
+                host: '34.211.174.194',
+                port: 26656,
+              },
+              {
+                id: 'd46e2445642b2f94158ac3c2a6d90b88b83705b8',
+                host: '3.76.148.150',
+                port: 26656,
+              },
+              {
+                id: 'b08a650ecfac178939f21c0c12801eccaf18a5ea',
+                host: '3.0.60.103',
+                port: 26656,
+              },
+              {
+                id: '4cb4a8488eb1dbabda7fb79e47ac3c14eec73c4f',
+                host: '188.166.140.47',
+                port: 26656,
+              },
+            ],
+          },
+          mempool: {
+            timeoutCheckTx: '3s',
+            txEnqueueTimeout: '30ms',
+            txSendRateLimit: 100,
+            txRecvRateLimit: 120,
+          },
+          genesis: {
+            chain_id: 'dash-mainnet-1',
```

Comment:
I believe this is put into every tenderdash block in the header, could we use something smaller?

Reply:
dashm would do.

Reply:
I believe verbosity is more important here

Reply:
@lklimek what will you say?

Reply:
`dash-1`?

Reply:
we can just do `dash-1`